eng/Directory.Build.Data.targets

```diff
@@ -10,15 +10,19 @@
     <Error Condition="'@(MissingTargetFrameworks)' != ''" Text="Missing required target frameworks '@(MissingTargetFrameworks)'. Please ensure you add those frameworks." />
   </Target>
 
-  <!-- Set PackageProjectUrl to the package README for Client Libraries -->
-  <Target Name="SetPackageProjectUrl" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true' and
+  <!-- Set PackageProjectUrl and PackageReleaseNotes to the package README.md and CHANGELOG.md respectively for DataPlane Libraries -->
+  <Target Name="SetPackageProjectUrlandReleaseNotes" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsDataPlaneProject)' == 'true' and '$(IsTestProject)' != 'true' and
    '$(IsSamplesProject)' != 'true' and '$(IsTestSupportProject)' != 'true' and '$(CommitSHA)' != ''">
     <PropertyGroup>
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
       <DirectoryPartofPath>$(CurrentCsprojPath.Substring($(CurrentCsprojPath.LastIndexOf("\sdk\"))))</DirectoryPartofPath>
       <DirectoryPartofPath>$(DirectoryPartofPath.Remove($(DirectoryPartofPath.LastIndexOf("\src"))))</DirectoryPartofPath>
       <PackageProjectUrl>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/README.md</PackageProjectUrl>
+      <PackageReleaseNotes Condition="'$(PackageReleaseNotes)' == '' and 'Exists('$(RepoRoot)/$(DirectoryPartofPath)/CHANGELOG.md')''">$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/CHANGELOG.md<PackageReleaseNotes>
```

Comment:
If you have cleaned out all the existing PackageReleaseNotes properties you should remove this part of the condition and instead and an error if a project sets that property.

Reply:
I removed PackageReleaseNotes  for Track 2 libraries. Some track one libraries still specify PackageReleaseNotes while some like servicebus want to use the changelog. I am trying to accommodate both cases.